### PR TITLE
Wait for Pod & Job to be ready

### PR DESCRIPTION
**Pull request by wgrzelak**
Job status:

```yaml
status:
  completionTime: 2019-02-11T15:36:54Z
  conditions:
  - lastProbeTime: 2019-02-11T15:36:54Z
    lastTransitionTime: 2019-02-11T15:36:54Z
    status: "True"
    type: Complete
  startTime: 2019-02-11T15:36:52Z
  succeeded: 1
```

Pod status:

```yaml
status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: 2019-02-11T16:51:01Z
    status: "True"
    type: Initialized
  - lastProbeTime: null
    lastTransitionTime: 2019-02-11T16:51:01Z
    message: 'containers with unready status: [command-demo-container]'
    reason: ContainersNotReady
    status: "False"
    type: Ready
  - lastProbeTime: null
    lastTransitionTime: 2019-02-11T16:51:01Z
    status: "True"
    type: PodScheduled
```

### CODE REVIEW

**Comment by wgrzelak** [comment]
Fixing python syntax in https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/pull/313

File: marketplace/deployer_util/wait_for_ready.py [hunk 1]
```diff
@@ -161,14 +185,33 @@ def is_ingress_ready(resource):
   return False
 
 
+def is_deployer_job(resource):
+  if 'app.kubernetes.io/component' in labels(resource):
+    return (labels(resource)['app.kubernetes.io/component'] ==
```
Comment (huyhg) [inline marketplace/deployer_util/wait_for_ready.py]: The UI doesn't currently populate this annotation. I will follow up with a CL to update it.

File: marketplace/deployer_util/wait_for_ready.py [hunk 2]
```diff
@@ -161,14 +185,33 @@ def is_ingress_ready(resource):
...
+def labels(resource):
+  return resource['metadata']['labels']
+
+
 def total_replicas(resource):
   return resource['spec']['replicas']
 
 
+def conditions_status(type, resource):
+  if 'conditions' in resource['status']:
```
Comment (huyhg) [inline marketplace/deployer_util/wait_for_ready.py]: Combine into one:

```python
for condition in resource.get('status', {}).get('conditions', []):
```

File: marketplace/deployer_util/wait_for_ready.py [hunk 3]
```diff
@@ -161,14 +185,33 @@ def is_ingress_ready(resource):
...
 
+def labels(resource):
+  return resource['metadata']['labels']
+
+
 def total_replicas(resource):
   return resource['spec']['replicas']
 
 
+def conditions_status(type, resource):
```
Comment (huyhg) [inline marketplace/deployer_util/wait_for_ready.py]: suggesting renames: `status_condition_is_true(condition_type, resource)`

(`type` is a reserved keyword in python)

Comment (wgrzelak) [reply marketplace/deployer_util/wait_for_ready.py]: This is nice !

File: marketplace/deployer_util/wait_for_ready.py (hunk 2 above)
Comment (wgrzelak) [reply marketplace/deployer_util/wait_for_ready.py]: This is nice!

**Comment by wgrzelak** [comment]
Huy, if the change is OK for you - can you merge the PR and release a new version?

Thanks in advance :)

**Pull request merged**